internal/controllers/cryostat_controller.go

```diff
@@ -316,6 +316,12 @@ func (r *CryostatReconciler) Reconcile(ctx context.Context, request ctrl.Request
 		}
 	}
 
+	instance.Status.GrafanaSecret = instance.Name + "-grafana-basic"
+	err = r.Client.Status().Update(context.Background(), instance)
```

Comment:
This should use `ctx` instead of `context.Background()`. We used `context.Background()` a lot before the Reconcile method provided a context to use, but I've been trying to replace this whenever making other changes to the affected code.